src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/repositories/DraftAdjudicationRepository.kt

```diff
@@ -9,4 +9,6 @@ interface DraftAdjudicationRepository : CrudRepository<DraftAdjudication, Long>
   fun findDraftAdjudicationByAgencyIdAndCreatedByUserIdAndReportNumberIsNull(agencyId: String, username: String): List<DraftAdjudication>
 
   fun deleteDraftAdjudicationByCreateDateTimeBeforeAndReportNumberIsNotNull(createdTime: LocalDateTime): List<DraftAdjudication>
+
+  fun findByCreateDateTimeAfterAndReportNumberIsNull(createdTime: LocalDateTime): List<DraftAdjudication>
```

Comment:
I don't think you should have this without knowing how it will be accessed.

Also I don't see why it looks for these - as it will include drafts that are being made. Actually why does he want to see drafts anyway?